src/components/Category.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Cartegory = () => (
+  <div>
+    <h1>Cartegory</h1>
+  </div>
```

Comment:
- [ ] For this project, you're to have a button in this component with the text Check Status. This button will do something in the future when clicked. Kindly add a button with the text Check Status  👍🏽

Reply:
Implemented the suggested changes.

Reply:
Add the button with check status content